lib/ex_unit/lib/ex_unit/capture_server.ex

```diff
@@ -75,7 +75,10 @@ defmodule ExUnit.CaptureServer do
     refs = Map.put(config.log_captures, ref, true)
 
     {level, opts} = Keyword.pop(opts, :level)
-    {formatter_mod, formatter_config} = Logger.default_formatter(opts)
+
+    {formatter_mod, formatter_config} =
+      Keyword.get_lazy(opts, :formatter, fn -> Logger.default_formatter(opts) end)
```

Comment:
According to the spec, `formatter` can be `{module(), term()} | nil`, but wouldn't this fail in the `nil` case?

Should it be `Keyword.get(opts, :formatter) || Logger.default_formatter(opts)`, or should we fix the spec?